cmake/qemu/CMakeLists.txt

```diff
@@ -158,8 +172,7 @@ if(EMU_PLATFORM)
       ${QEMU_FLAGS}
       ${QEMU_EXTRA_FLAGS}
       ${MORE_FLAGS_FOR_${target}}
-      -kernel $<TARGET_FILE:${logical_target_for_zephyr_elf}>
-      DEPENDS ${logical_target_for_zephyr_elf}
```

Comment:
We should not be removing this DEPENDS line.

Reply:
I think keeping DEPENDS will fix this issue:

>make run right away for a Jailhouse build (cmake -DBOARD=x86_jailhouse -DJAILHOUSE_QEMU_IMG_FILE=something) on the synchronization example still does not compile things before firing up qemu.

Reply:
Sorry, merge oopsie. Will fix it.